src/gis_utils/dapla.py

```diff
@@ -1,9 +1,10 @@
 import geopandas as gpd
 import numpy as np
 import pandas as pd
+from pyarrow import parquet
 
```

Comment:
Importer skal være i toppen av file, i følge PEP8. Legg til:
-  `from dapla import Fileclient, details`
- `import os`

Reply:
@mortewle Ja, enig i det. Spørsmålet er egentlig om funksjonene read_geopandas og write_geopandas trenger å skrive til bøtter på dapla? Hvis ja, så må avhengigheten være der, siden koden bruker dapla-toolbelt og ikke vil virke uten at dapla-toolbelt er importert. Man kan alternativt ha det som en extras dependency, ala `poetry install ssb-gis-utils[dapla]` som beskrevet [her](https://python-poetry.org/docs/pyproject/#extras).

Et annen ting å vurdere er om koden kanskje burde sjekke om den kjører på dapla eller ikke (jeg har kode for dette), og skrive til vanlig filsti hvis den ikke kjører på dapla, og skrive til dapla-bøtter hvis den gjør det. Vi kan diskutere dette mer i morgen, og la det det være slik det er nå inntil videre.